bin/pos-cli-data-export.js

```diff
@@ -36,36 +37,56 @@ const fetchFilesForData = async data => {
 program
   .name('pos-cli data export')
   .arguments('[environment]', 'name of the environment. Example: staging')
-  .option('-p --path <export-file-path>', 'output for exported data', 'data.json')
+  .option('-p --path <export-file-path>', 'output for exported data. Example: data.json, data.zip', 'data.json')
```

Comment:
This will cause confusion

I think it would be just easier, to compress file into data.json.zip if user did `-z` and not mention it here at all

Reply:
Makes sense, on it.

Reply:
On second thought this might be confusing, as the zip archive is not a single json file, but several.

Reply:
Just a note here that `data.json` and `data.zip` are in a completely different format. ZIP has new importer's CSV files inside. data.json has an old JSON format. We are going to deprecate old import and new one was supposed to be added here. So, we can use whatever we want for the interface, but we have to remember those are different things.

Reply:
Is there a consensus around this topic?
I believe best would be to live it just like this for now, as later we will remove .json support and .zip will be the default.

Reply:
I'm assuming this is resolved and we can go ahead and merge. One thing that could cause this confusion is the name/description(lack of thereof) of this PR. We should probably do better next time, explaining this ZIP is with the new CSV file format in it 😉